lib/src/date-input/DateInput.stories.tsx

```diff
@@ -110,42 +113,96 @@ export const Chromatic = () => (
+    <DxcDateInput label="Date input" defaultValue="06-04-1950" />
   </ExampleContainer>
 );
 
-export const ShowYearPicker = YearPicker.bind({});
+export const ShowYearPicker = YearPickerCalendar.bind({});
+ShowYearPicker.parameters = {
+  date: new Date("1947-12-03"),
+};
 ShowYearPicker.play = async ({ canvasElement }) => {
   const canvas = within(canvasElement);
   await userEvent.click(canvas.getByRole("combobox"));
-  await fireEvent.click(screen.getByText("April 2027"));
+  await fireEvent.click(screen.getByText("April 1950"));
+  await fireEvent.keyDown(document.activeElement, {
+    key: "ArrowUp",
+    code: "ArrowUp",
+    keyCode: 38,
+    charCode: 38,
+  });
 };
 
-const YearPickerFocus = () => (
-  <ExampleContainer expanded>
-    <Title title="Show year picker and focus" theme="light" level={4} />
-    <DxcDateInput label="Date input" defaultValue="06-04-2027" />
-  </ExampleContainer>
+const DatePickerCalendar = () => (
+  <>
+    <ExampleContainer expanded>
+      <Title title="Isolated calendar" theme="light" level={4} />
+      <DxcDatePicker date={dayjs("06-04-1950", "DD-MM-YYYY")} onDateSelect={() => {}} id="test-calendar" />
+    </ExampleContainer>
+    <ExampleContainer expanded pseudoState="pseudo-focus">
+      <Title title="Isolated calendar focused" theme="light" level={4} />
+      <DxcDatePicker date={dayjs("06-04-1950", "DD-MM-YYYY")} onDateSelect={() => {}} id="test-calendar" />
+    </ExampleContainer>
+    <ExampleContainer expanded pseudoState="pseudo-hover">
+      <Title title="Isolated calendar hovered" theme="light" level={4} />
+      <DxcDatePicker date={dayjs("06-04-1950", "DD-MM-YYYY")} onDateSelect={() => {}} id="test-calendar" />
+    </ExampleContainer>
+    <ExampleContainer expanded pseudoState="pseudo-active">
+      <Title title="Isolated calendar actived" theme="light" level={4} />
+      <DxcDatePicker date={dayjs("06-04-1950", "DD-MM-YYYY")} onDateSelect={() => {}} id="test-calendar" />
+    </ExampleContainer>
+  </>
 );
+export const ShowIsolatedCalendar = DatePickerCalendar.bind({});
+ShowIsolatedCalendar.parameters = {
+  date: new Date("1950-04-03"),
+};
 
-export const ShowYearPickerFocus = YearPickerFocus.bind({});
-ShowYearPickerFocus.play = async ({ canvasElement }) => {
-  const canvas = within(canvasElement);
-  await userEvent.click(canvas.getByRole("combobox"));
-  await fireEvent.click(screen.getByText("April 2027"));
-  await userEvent.tab();
+const YearPickerIsolated = () => (
+  <>
+    <ExampleContainer expanded>
+      <Title title="Isolated year picker" theme="light" level={4} />
+      <YearPicker selectedDate={dayjs("06-04-1905", "DD-MM-YYYY")} onYearSelect={() => {}} />
+    </ExampleContainer>
+    <ExampleContainer expanded pseudoState="pseudo-focus">
+      <Title title="Isolated year picker focused" theme="light" level={4} />
+      <YearPicker selectedDate={dayjs("06-04-1905", "DD-MM-YYYY")} onYearSelect={() => {}} />
```

Comment:
The focus is a bit cut.
![image](https://user-images.githubusercontent.com/34685461/214272825-ff104342-2763-457a-a008-b760bfde0e6d.png)